equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/mapper/Crm5Mapper.java

```diff
@@ -125,4 +125,15 @@ public interface Crm5Mapper {
     @Mapping(target="solicitor.certification.name", source="certification_sol_name")
     @Mapping(target="officeUseOnly.decision", source="decision")
     CRM5DetailsDTO getDetailsDTOFromModel(Crm5DetailsModel crm5DetailsModel);
+
+    default String processAcPreparationTime(Crm5DetailsModel crm5DetailsModel) {
+        if (null != crm5DetailsModel.getAc_preparation_time()) {
```

Comment:
this function does not need to be a default one.
Maybe we can have a generic: convertToTimeSpentString() and move it to the interface and have it as a generic there